Cargo.lock

```diff
@@ -840,6 +840,7 @@ dependencies = [
  "http",
  "k8s-openapi",
  "kube",
+ "maplit",
```

Comment:
For the lack of a better place for this comment: In the commit message replace any `customer` occurrence with `user`.